### ISSUE TITLE
loki plugin getUser response is not SCIM compliant

### ISSUE THREAD

**Issue by mohitzim** [jelhub/scimgateway#18]
Hi jelhub

I am using loki plugin and doing filter request as following :
https://domain/Users?filter=userName+eq+%22bjensen1%22

It works fine when user is found, but when user is not found, its sending 404 response instead of 200 with empty array.
Azure AD expects following response when user is not found :
```
{
	"schemas": ["urn:ietf:params:scim:api:messages:2.0:ListResponse"],
	"totalResults": 0,
	"Resources": [],
	"startIndex": 1,
	"itemsPerPage": 20
}
```
Can you please explain regarding this.

Thanks & Regards

**Comment by jelhub** [comment]
Hi,
Azure AD as IdP requires SCIM 2.0 to be used by plugin.  

	"scimgateway": {
	  ...
	  "scim": {
		"version": "2.0"
		...

Also make sure you are running latest version (v2.1.4)
Problem reported was fixed in v2.0.2

Regards,
Jarle

**Comment by mohitzim** [comment]
Thanks @jelhub. This worked with version change in config.
